scrapinghub/client/utils.py

```diff
@@ -140,6 +132,46 @@ def list(self, *args, **kwargs):
         return list(self.iter(*args, **kwargs))
 
 
+class _MappingProxy(_ProxyBase):
+
+    def __init__(self, cls, client, key):
+        super(_MappingProxy, self).__init__(cls, client, key)
+        methods = ['ignore_fields', 'expire', 'save', 'liveget']
+        self._proxy_methods(methods)
+        self.iterkeys = self.__iter__
```

Comment:
I wonder, maybe we can ask _MappingProxy to map everything *except* some methods? I think it'll be a short list